### ISSUE TITLE
Warning when using Nuxt

### ISSUE THREAD

**Issue by roeeyn** [EducationLink/vue-tel-input#73]
Hi, I'm using Nuxt 2.0 and I had been using vue-tel-input since version 2.0.10 without any problem. But recently I updated it to version 2.0.18 and it started to throw the following warning:

```
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

As far as I know, it is an issue with webpack itself but is there something I can do?

**Comment by iamstevendao** [comment]
Might be because of updating `serve` package in 458c8b230e48e9e3f18c9864e3fe088aa40da08c?
Do you have an easy way to reproduce your issue? I will make some tests.

**Comment by roeeyn** [comment]
Just import it as show in the examples, it will throw the warning. Thanks for the help 😄

**Comment by Faisal0sal** [comment]
I'm running nuxt 2 and it's working for me, are you sure your setup is right?

**Comment by roeeyn** [comment]
My setup is the following one:
I have my plugin declared in my puglin folder ```~/plugins/vue-tel-input.js```:
![plugin screenshot](https://res.cloudinary.com/dfeprzkue/image/upload/v1555276001/GitHub-vue-tel-input/gh-1.png)

Then I have it declared in my ```~/nuxt.config.js```
![nuxt config ss](https://res.cloudinary.com/dfeprzkue/image/upload/v1555276000/GitHub-vue-tel-input/gh-3.png)

Finally I use it in my component:
![use ss](https://res.cloudinary.com/dfeprzkue/image/upload/v1555276001/GitHub-vue-tel-input/gh-2.png)

As I said before, this worked totally fine until the update of the library. I used nuxt 2 too by te way.

Thanks for the help 😄

**Comment by iamstevendao** [comment]
@roeeyn We changed the way we built in bbadda59b1628145d17169f9352c358abcf4a6c1. Can you update to the newest version and see if it's still happening?

**Comment by iamstevendao** [comment]
I think it should be fixed in the same way with #105. Feel free to re-open if it still happens to you.
